src/Simplex/Messaging/Agent/Client.hs

```diff
@@ -285,7 +285,7 @@ sendHello c sq@SndQueue {server, sndId, sndPrivateKey} verifyKey RetryInterval {
           let newDelay =
                 if elapsedTime < increaseAfter || delay == maxInterval
                   then delay
-                  else min (delay * 3 `div` 2) maxInterval
+                  else min (delay * 20 `div` 19) maxInterval
```

Comment:
why the multiple is so close to 1 now? I think it would hammer it too much in async case.